### PR TITLE
[Labware] Add several plates

### PR DESCRIPTION
**Pull request by fderop**
Added the following:
- Falcon_96_WP_Fl
- Falcon_96_WP_Rb
- Falcon_96_WP_Fl_Black
- Corning_6_WP_Fl
- CellTreat_96_WP_U
- AGenBio_4_WP_V
- AGenBio_1_WP_Fl
- Porvair_24_WP_V

### CODE REVIEW

File: pylabrobot/resources/corning_costar/plates.py [hunk 1]
```diff
@@ -12,8 +13,10 @@
   calculate_liquid_height_container_1segment_round_fbottom,
   calculate_liquid_height_in_container_2segments_square_vbottom,
   calculate_liquid_volume_container_1segment_round_fbottom,
-  calculate_liquid_volume_container_2segments_square_vbottom
+  calculate_liquid_volume_container_2segments_square_vbottom,
+  compute_volume_from_height_conical_frustum
 )
+from retro_pylabrobot.resources.plates import _compute_height_from_volume_conicalfrustum
```
Comment (rickwierenga) [inline pylabrobot/resources/corning_costar/plates.py:19]: this isn't in PLR yet

Comment (fderop) [reply pylabrobot/resources/corning_costar/plates.py]: Fixed

**Comment by rickwierenga** [comment]
following the naming convention: https://github.com/PyLabRobot/pylabrobot/pull/156

**Comment by rickwierenga** [comment]
nice, great to be open sourcing more!

**Pull request merged**